frontend/src/components/VideoExamination/ExportAnnotations.vue.js

```
@@ -0,0 +1,253 @@
+                toast.error({ text: `Segment ${segment.id} konnte nicht aktualisiert werden.` });
+                break;
+            }
+            updatingSegments.value.delete(segment.id);
+        }
+    }
+    finally {
+        updatingSegments.value.clear();
+        isBulkUpdating.value = false;
+    }
+};
+debugger; /* PartiallyEnd: #3632/scriptSetup.vue */
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)